tests/test_reportlog.py

```diff
@@ -47,13 +47,16 @@ def test_fail():
 
     # rest of the json objects should be unserialized into report objects; we don't test
     # the actual report object extensively because it has been tested in ``test_reports``
-    # already.
+    # already (except start and stop attributes which are set by this plugin).
```

Comment:
This edge case doesn't sound good though. Up to this point the only responsibility of this plugin was saving reportlog to a json file as is.

I don't know where this new feature fits best, so I created this pull request to start off the discussion.
Alternatively I can try to upstream it to pytest itself, here's a draft - https://github.com/HTRafal/pytest/pull/1

Let me know what you guys think.

Reply:
As commented in https://github.com/HTRafal/pytest/pull/1#issuecomment-1414107011, seems like it would be a good fit to move the timestamps directly to the test report objects. :+1:

Reply:
Got it, I opened a PR to core pytest instead - https://github.com/pytest-dev/pytest/pull/10711